v2/charts/azure-service-operator/values.yaml

```diff
@@ -92,6 +92,25 @@ metrics:
 # the operator will only reconcile that subset if those are the only CRDs it finds when the pod starts.
 installCRDs: true
 
+# crdPattern is a semicolon delimited string containing the CRD patterns for the operator to install.
+# Setting this has no effect if installCRDs is false.
+# This defines what new CRDs will be installed by ASO. Will always upgrade any existing CRDs even if no
+# crdPattern is defined. Leaving this field unspecified for a fresh install of ASO will result in the operator pod
+# exiting with an error saying no CRDs are configured. Leaving this field unspecified during an upgrade of ASO preserves
+# the existing set of CRDs already in the cluster. The existing CRDs will be upgraded to the latest version but no
+# new CRDs will be installed.
+# Values can be globs utilizing * or ?. The pattern is compared against the "{group}/{kind}" string of each CRD.
```

Comment:
What about making `/*` optional and allowing a pattern that's just `{group}` as well?

Reply:
I think there's value in it being "a glob" because people at least in theory understand that from shell-days. If we did this it would be "a glob but with some exceptions"

Maybe if there's actually user confusion or complaints about this we could revisit? I think for now though the simpler "it's a glob" has the advantage of being both consistent and also implemented in this PR. We would need to change things up possibly significantly in order to allow non-glob shortcuts as it's not clear what `matcher` would make sense with them, so we'd probably have to filter them out first and then handle them outside of the `matcher` context?